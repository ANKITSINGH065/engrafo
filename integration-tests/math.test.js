var utils = require("./utils");

test("block math renders correctly", done => {
  utils.expectBodyToMatchSnapshot("math/block.tex", done);
});

test("inline math renders correctly", done => {
  utils.expectBodyToMatchSnapshot("math/inline.tex", done);
});
