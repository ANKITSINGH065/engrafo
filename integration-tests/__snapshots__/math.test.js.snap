// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block math renders correctly 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    <dt-article>
      <p />
      <figure
        id="equation-1"
      >
        <p>
          <span
            class="math display"
          >
            \\[e^{i\\pi}=-1 \\]
          </span>
          <span
            class="engrafo-equation-number"
          >
            (1)
          </span>
        </p>
      </figure>
      <p />
      <p>
        <a
          href="#equation-1"
        >
          Equation 1
        </a>
        is some good math.
      </p>
    </dt-article>
    <dt-appendix>
      <div
        class="l-body"
      />
    </dt-appendix>
    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;

exports[`inline math renders correctly 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    <dt-article>
      <p>
        Ooh look some lovely maths 
        <span
          class="math inline"
        >
          \\(x^2 + y^2 = z^2\\)
        </span>
        oh yeah.
      </p>
    </dt-article>
    <dt-appendix>
      <div
        class="l-body"
      />
    </dt-appendix>
    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;
